conans/model/layout.py

```diff
@@ -90,7 +90,7 @@ def source_folder(self):
         if self._base_source is None:
             return None
         if not self.source:
-            return self._base_source
+            return os.path.normpath(self._base_source)
```

Comment:
Is this pattern repeated elsewhere?

Reply:
It's fixed for all the cases found here.